packages/native/types/tests.tsx

```diff
@@ -23,9 +23,9 @@ declare module '@emotion/react' {
 const cssObject = {
   height: 100,
   width: '100%',
-  display: 'flex',
+  display: 'flex' as const,
```

Comment:
q: why the change here?

Reply:
dtslint complained that `as const` should never be used with object literals. The `as const` is necessary since `display` must have type `'flex' | 'none' | undefined`. So I just moved it to make dtslint happy.

Reply:
> dtslint complained that as const should never be used with object literals.

That's weird :o was this perhaps this rule that has been complaining: https://palantir.github.io/tslint/rules/no-object-literal-type-assertion/ ? quite bizarre that it would disallow `as const` 🙈 

I don't mind the change though